src/components/Footer.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div id="footer">
```

Comment:
wouldn't be better to use the `footer` element here instead of a div?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer